infrastructure/mainTemplate.json

```diff
@@ -523,7 +527,7 @@
                     "appSettings": [
                         {
                             "name": "AAS_LIST_FEATURE_FLAG",
-                            "value": "true"
+                            "value": "false"
```

Comment:
I would prefer to set `Features__AllowRetrievingAllShellsAndSubmodels` to true instead of deactivating the AAS list

Reply:
I get what you mean, but I don't think people are unaware, that everyone with access to the repository can see everything. I think it's more of a niche case that people don't want to see everything and would therefore prefer to have the AAS list visible by default